scipy/sparse/csgraph/_reordering.pyx

```diff
@@ -197,12 +197,17 @@ def structural_rank(graph):
     -------
     rank : int
         The structural rank of the sparse graph.
-    
+
+    Notes
+    -----
+    Algorithmic complexity is :math:`O(\lvert E \rvert \sqrt{\lvert V \rvert})`,
```

Comment:
I don't think you need the `\lvert` here if V is `the number of vertices`. That would be for V being the set of vertices.  BTW: I'm glad to see that you are stating what E and V are.

Reply:
Ah! yes correct. Thanks for pointing out. My bad.